res/controllers/Reloop Beatpad.midi.xml

```diff
@@ -1153,8 +1133,18 @@
                 </options>
             </control>
             <control>
+                <group>[Channel2]</group>
+                <key>ReloopBeatpad.spflBtn</key>
+                <description>SHIFT+PFL 2</description>
+                <status>0x91</status>
+                <midino>0x21</midino>
```

Comment:
it's consisten with SHIFT+PFL 1 (90/21) , PFL1(90/41)  and PFL2 (91/41)

channel 1 --> 0x90 / channel 2 --> 0x91
shifted control = control - 0x40